google-cloud-pubsub/lib/google/cloud/pubsub/subscriber/timed_unary_buffer.rb

```diff
@@ -60,6 +60,17 @@ def modify_ack_deadline deadline, ack_ids
           end
           alias delay modify_ack_deadline
```

Comment:
Is delay (or indeed modify_ack_deadline) called at all now?

Reply:
Yes. `ReceivedMessage#modify_ack_deadline!` and `ReceivedMessage#reject!` still call this when the received message is pulled using a `Subscriber`.

Reply:
ok